controller/api/polygon.js

```diff
@@ -166,6 +166,7 @@ class PolyCtrl {
                     logFilePath: LOG_FILES_PATH_NAMES.wrapPolygonTransactionQueue,
                     jobIsRunningCacheKey: ORACLE_CACHE_KEYS.isWrapOnPolygonJobExecuting
                 })
+                console.log(logPrefix + 'Oracle data is not walid, pls check .env and contract abi.')
```

Comment:
Could we create default Oracle message somewhere and use it for `ETH` and `POLYGON`?

Reply:
not necessery for now